code/ShareCareFields.php

```diff
@@ -21,7 +21,7 @@ class ShareCareFields extends DataExtension
      * @var string
      * @config
      */
-    private static $cms_message = 'The preview above is automatically generated from your content. You can override the default values using these fields:';
+    private static $cms_message = _t('ShareCareFields.CMSMessage');
```

Comment:
As with previous, please remove this static and reference the translatable string instead in code.